app/change_set_persisters/hyrax/change_set_persister.rb

```diff
@@ -32,16 +34,35 @@ def save_all(change_sets:)
       end
     end
 
-    def buffer_into_index
-      metadata_adapter.persister.buffer_into_index do |buffered_adapter|
-        with(metadata_adapter: buffered_adapter) do |buffered_changeset_persister|
-          yield(buffered_changeset_persister)
-        end
+    def delete_all(change_sets:)
```

Comment:
Is this a public API? Can it be documented?

Reply:
Doc Added